src/System.IdentityModel.Tokens/project.json

```diff
@@ -3,12 +3,18 @@
     "dependencies": {
         "Newtonsoft.Json": "6.0.6"
     },
+    "exclude": [
+        "Saml\\SamlSecurityTokenHandler.cs",
+        "Saml\\Saml2SecurityTokenHandler.cs",
+        "Saml\\SamlConstants.cs",
+        "Saml\\SecurityKeyResolver.cs",
+        "Tokens\\SecurityTokenHandlerCollectionExtensions.cs"
```

Comment:
Where do the SamlSecurityToken.cs and Saml2SecurityToken.cs classes live? Why they are excluded in original project.json but not in S.IM.Tokens project.json?